tests/transforms/test_optimization/test_merge_rotations.py

```diff
@@ -424,13 +424,11 @@ def test_merge_rotations_jax_jit(self):
         @qml.qnode(qml.device("default.qubit", wires=["w1", "w2"]), interface="jax")
         @merge_rotations
         def qfunc():
-            qml.Rot(jax.numpy.array(0.1), jax.numpy.array(0.2), jax.numpy.array(0.3), wires=["w1"])
-            qml.Rot(
-                jax.numpy.array(-0.1), jax.numpy.array(-0.2), jax.numpy.array(-0.3), wires=["w1"]
-            )
+            qml.Rot(*jax.numpy.array([0.1, 0.2, 0.3]), wires=["w1"])
+            qml.Rot(*jax.numpy.array([-0.3, -0.2, -0.1]), wires=["w1"])
```

Comment:
This test case was falsely verifying the simple addition of `Rot` angles, it's doing its job now.